src/ios-deploy/ios-deploy.m

```diff
@@ -91,6 +91,7 @@
 mach_error_t AMDeviceLookupApplications(AMDeviceRef device, CFDictionaryRef options, CFDictionaryRef *result);
 int AMDeviceGetInterfaceType(AMDeviceRef device);
 AMDeviceRef AMDeviceCopyPairedCompanion(AMDeviceRef device);
+bool AMDeviceCopyDeveloperModeStatus(AMDeviceRef device, unsigned int *arg1);
```

Comment:
Got some build issue for this

```
Undefined symbols for architecture arm64:
  "_AMDeviceCopyDeveloperModeStatus", referenced from:
      _check_developer_mode in ios-deploy.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

relates to https://github.com/Homebrew/homebrew-core/pull/104730

Reply:
Oops, this symbol is new so Xcode 14 is required to build it. Will revert this commit and look into gating this to only be included when building with Xcode 14.

Reply:
Tested out the revert, works for me in my local. Thanks @ivanhernandez13!